sklearn/ensemble/forest.py

```diff
@@ -313,11 +313,9 @@ def fit(self, X, y, sample_weight=None):
                 # would have got if we hadn't used a warm_start.
                 random_state.randint(MAX_INT, size=len(self.estimators_))
 
-            trees = []
-            for i in range(n_more_estimators):
-                tree = self._make_estimator(append=False,
-                                            random_state=random_state)
-                trees.append(tree)
+            trees = [
```

Comment:
We generally prefer visual indentation such as:
```py
            trees = [self._make_estimator(append=False,
                                          random_state=random_state)
                     for i in range(n_more_estimators)]
```
(This may not always be preferable when there is a lot to the left of `[`, but a bare `= [` looks ugly)

Reply:
>  but a bare = [ looks ugly)

Updated as suggested, though using https://github.com/ambv/black might change your mind about this :)